flask_app/src/query/init_layer_2.sql

```diff
@@ -24,3 +24,11 @@ VALUES (
 -- Metadata
 INSERT INTO "metadata" (meta_value_id, asset_id)
 VALUES (1, 1), (7, 1), (11, 1), (12, 1);
+
+
+-- Comment
+INSERT INTO "comment" (content, created_at, external_name, posted_by, asset_id)
+VALUES (
+        'Sed ut perspiciatis unde omnis iste natus error sit voluptatem translated',
+        CURRENT_TIMESTAMP, '', 1, 1
+       );
```

Comment:
```suggestion
VALUES 
       (
        'Sed ut perspiciatis unde omnis iste natus error sit voluptatem translated',
        CURRENT_TIMESTAMP, NULL, 1, 1
       ),
       (
        'MAKE SALINE GREAT AGAIN !!!',
        CURRENT_TIMESTAMP, "Donald TRUMP", NULL, 1
       );
```
Il faudrait réfléchir sur le fait d'avoir un comment d'une personne externe sans qu'il soit dans notre base User

Reply:
Yes c'est super attendu par les profs en plus. Faudra se synchro avec le front également pour voir ce qu'ils prévoient

Reply:
Disons que c'est nettement plus propre d'avoir des valeurs NULL plutôt que des string vides (j'en ai fait l'expérience cette semaine en entreprise en utilisant une API externe...)